src/views/InteractionDesigner.vue

```diff
@@ -234,6 +234,9 @@ export default {
       if (field) {
         scrollBehavior(this.$route)
       }
+      if (this.$route.name === 'flow-simulator' && this.hasSimulator()) {
```

Comment:
What if we move this in `updated()` hook, so can consider `isSimulatorActive` as the single truth.
Having route.name test in template seem quite confusing at the 1st glance. I'd love to have `<div v-if="isSimulatorActive" class="tree-sidebar">` (from
 https://github.com/FLOIP/flow-builder/pull/75/files#diff-4778731e3debd774a6c14025179b9bfb34aae3d45fb42448205377ca6475503dR169). 
 Your thoughts ?